### PR TITLE
Update example_career_path.md

### PR DESCRIPTION
**Pull request by ljgarcia**
Some style- and grammar-related changes. The message remains the same.

### CODE REVIEW

File: example_career_path.md [hunk 1]
```diff
@@ -7,63 +7,67 @@
...
-problems makes her happier than the subdued sense of achievement from having a research paper accepted long after she had written it.
-
-The generic RSE Master programme consists of a core of RSE topics with various electives for specialisation.
-Kay chooses digital archaeology and develops a pipeline for reconstructing 3D models from ground penetrating radar data, to simplify the process for archaeologists (Reproducibility, Big Data, ML).
-The project management skills that are being taught as part of the core RSE curriculum really help her to not get lost in this project.
-Apart from working with the researchers in her archaeology group she has to work with members of the central RSE department to help her with the pipelines. 
+She also experienced the immediate satisfaction gained by helping colleagues from her research group with tricky technical
+problems, which makes her happier than the subdued sense of achievement from having a research paper accepted long after she had written it.
+For her, coding and sharing knowledge in the form of software is more exciting than writing a paper focused mostly on the obtained results.
```
Comment (jlinx) [inline example_career_path.md:30]: Little provocative but I like the idea. However, I don't think the question is if she likes one or the other better. But, she likes the second (software publication option), too.

Comment (jlinx) [inline example_career_path.md]: Little provocative but I like the idea. However, I don't think the question is if she likes one or the other better. But, she likes the second (software publication option), too.

```suggestion
However, for her, coding and sharing knowledge in the form of software is of similar importance to writing a paper focused mostly on the obtained results.
```

File: example_career_path.md [hunk 2]
```diff
@@ -7,63 +7,67 @@
...
+And to keep track of where the project is going and how to distribute her time, she leverages Kanban boards (PM skills).
 Her work so far has already been heavy on (TEAM) skills, but now also the leadership aspect comes into play.
+
+## RSE-focused Principal Investigator
+
 The job experience as a leading RSE for a large project was the last requirement necessary to be awarded the "Science Enabling Doctor" from an
 institutionalised centre of RSE education, recognising her track record of valuable software contributions, and her specialised experience in
 leading and mentoring people. Some personal choices in her career made the traditional route of doing a Dr.-Ing. inconvenient.
-These doctoral degrees are of course recognised by various funding agencies, such as the DFG, and hence enables RSEs that have chosen to not follow a research focused
+These doctoral degrees are of course recognised by various funding agencies, such as the DFG, and hence enables RSEs that have chosen not to follow a research focused
```
Comment (jlinx) [inline example_career_path.md:68]: Not so sure about this one.

```suggestion
These doctoral degrees are of course recognised by various funding agencies, such as the DFG, and hence enables RSEs that have chosen to not follow a research focused
```

**Review by jlinx — COMMENTED** [review]
Some minor changes

File: example_career_path.md (hunk 2 above)
Comment (jpthiele) [reply example_career_path.md:68]: I'm also not sure. 
@jcohen02 any gut feelings on this as a native speaker?

File: example_career_path.md [hunk 3]
```diff
@@ -7,63 +7,67 @@
...
-Kay chooses digital archaeology and develops a pipeline for reconstructing 3D models from ground penetrating radar data, to simplify the process for archaeologists (Reproducibility, Big Data, ML).
-The project management skills that are being taught as part of the core RSE curriculum really help her to not get lost in this project.
-Apart from working with the researchers in her archaeology group she has to work with members of the central RSE department to help her with the pipelines. 
+She also experienced the immediate satisfaction gained by helping colleagues from her research group with tricky technical
+problems, which makes her happier than the subdued sense of achievement from having a research paper accepted long after she had written it.
+However, for her, coding and sharing knowledge in the form of software is of similar importance to writing a paper focused mostly on the obtained results.
+
+The domain-agnostic RSE Master programme consists of a core of RSE topics with various electives for specialisation, some of them domain-specific (e.g., chemistry) or topic-specifi (e.g., cloud computing for reserach).
```
Comment (CaptainSifff) [inline example_career_path.md:32]:
```suggestion
The domain-agnostic RSE Master programme consists of a core of RSE topics with various electives for specialisation, some of them domain-specific (e.g., chemistry) or topic-specific (e.g., cloud computing for reserach).
```

File: example_career_path.md [hunk 4]
```diff
@@ -7,63 +7,67 @@
...
 Kay finds a position in the central RSE department at her university with a competitive IT salary.
-Although the contract is temporary there is a good chance that it will lead to a permanent position.
+Although the contract is temporary, there is a good chance that it will lead to a permanent position.
 Kay completes the Software Carpentry Instructor training and teaches basic research computing, 
 while advising fellow students of her department on better programming (DOCBB and MOD skill)
 She also runs a seminar in the RSE Master programme. She publishes a condensed version of that in JOSE[@JOSE].
-During her teaching duties, she becomes aware of a new project in her department that requires a community managing RSE
-and she gladly signs to focus more on her communication skills.
-After two years she takes an exciting opportunity to work in another university.
+During her teaching duties, she becomes aware of a new project in her department that requires an RSE community manager,
```
Comment (CaptainSifff) [inline example_career_path.md:49]: Ah, OK, that's tricky....: What I meant was: an RSE that takes on the responsibility of managing a community (e.g. of domain scientist users).
I understand your correction, to mean a manager that manages a community of RSEs.

**Review by CaptainSifff — CHANGES_REQUESTED** [review]
lloks perfect! thanks for the improvement! apart from one raised question we can merge it!

File: example_career_path.md (hunk 2 above)
Comment (ljgarcia) [reply example_career_path.md:68]: As far as I understand, "not to" is more formal than "to not".

Comment (jcohen02) [reply example_career_path.md:68]: I guess I would say that "not to" is more definite (and I suppose more formal) in that I think "to not" is perhaps a more relaxed way of saying the same thing - so yes, I agree with @ljgarcia, I think classing one as more formal than the other is correct.

File: example_career_path.md (hunk 4 above)
Comment (CaptainSifff) [reply example_career_path.md:49]: Here we need to decide which viewpoint to take.

Comment (CaptainSifff) [reply example_career_path.md:49]: maybe I should stick to the original "community manager RSE"

**Comment by CaptainSifff** [comment]
Oh, I'm so sorry.... This is the wrong file. The content has been merged into the main file competencies.md
I'll see which of your fixes still persist.

**Pull request merged**